typescript/sdk/src/hook/EvmHookModule.hardhat-test.ts

```diff
@@ -156,15 +153,19 @@ function randomHookConfig(
 }
 
 describe('EvmHookModule', async () => {
+  const chain = TestChainName.test4;
+
+  // beforeEach
   let multiProvider: MultiProvider;
   let coreAddresses: CoreAddresses;
-
-  const chain = TestChainName.test4;
+  let fundingAccount: Signer;
   let proxyFactoryAddresses: HyperlaneAddresses<ProxyFactoryFactories>;
   let factoryContracts: HyperlaneContracts<ProxyFactoryFactories>;
+  let exampleRoutingConfig: DomainRoutingHookConfig | FallbackRoutingHookConfig;
 
   beforeEach(async () => {
-    const [signer] = await hre.ethers.getSigners();
+    const [signer, funder] = await hre.ethers.getSigners();
+    fundingAccount = funder;
```

Comment:
set `let signer: Signer;` in parent scope and

```suggestion
    [signer, funder] = await hre.ethers.getSigners();
```